projects/ecoindex_cli/README.md

```diff
@@ -28,6 +28,14 @@ Then you can use the cli as if it was installed on your computer:
 ecoindex-cli --help
 ```
 
+If you are on linux and you want to perform analysis on local project you need to add the ```--add-host=host.docker.internal:host-gateway``` flag to the docker run command as follow :
```

Comment:
**❓ question(non-blocking):** Only for linux ?

Reply:
Yes

Reply:
OK, so for mac and windows, no need to add this line ?

Reply:
Nope

Reply:
OK, so maybe for the dumb people like me we can propose 2 aliases:
- One for linux users
- Another for windows and mac users
?

Reply:
Done